analytics/pubxai/config/mocks/config.go

```diff
@@ -0,0 +1,63 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Can you use the testify mock library? That's our standard in this project.

Reply:
sure, updated all the mocks to use testify mock